azblob/zc_policy_request_log.go

```diff
@@ -110,7 +110,7 @@ func NewRequestLogPolicyFactory(o RequestLogOptions) pipeline.Factory {
 }
 
 // redactSigQueryParam redacts the 'sig' query parameter in URL's raw query to protect secret.
```

Comment:
It would be good to update the comments for doc purpose if wish to expose it as public method.

Reply:
Done